bounties/pip/BentoML/1/README.md

```diff
@@ -0,0 +1,25 @@
+# Description
+BentoML is a framework for serving, managing, and deploying machine learning models. It is aiming to bridge the gap between Data Science and DevOps, and enable teams to deliver prediction services in a fast, repeatable, and scalable way.
+* **Vulnerability discription**
+    untrusded loading of data by the picle.load function leading to Arbitrary code execution.
+# Proof of Concept
+
+1. run exploit.py
+```python
+// exploit.py
```

Comment:
Invalid in `python`: comments made with `#`

Reply:
changed ..